### PR TITLE
corrected directory paths

### PR DESCRIPTION
**Pull request by parteekcoder**
issue #368

### CODE REVIEW

**Comment by llaske** [comment]
Please @parteekcoder, do not submit a PR until we've reproduce ypur issue. We need to ensure that's really an issue.

File: env/sugarizer.ini [hunk 1]
```diff
@@ -22,7 +22,7 @@ consent_need = false
 policy = https://sugarizer.org/policy.html
 
 [client]
-path = ../sugarizer/
```
Comment (NikhilM98) [inline env/sugarizer.ini:25]: The client app is Sugarizer, not Sugarizer-Server.

Comment (parteekcoder) [reply env/sugarizer.ini:25]: but when tried ran using this it gives directory not found

when I ran with my changes it ran fine

Comment (parteekcoder) [reply env/sugarizer.ini:25]: @NikhilM98 see this

image

Comment (NikhilM98) [reply env/sugarizer.ini:25]: Sugarizer has a directory named `activities`: https://github.com/llaske/sugarizer/tree/master/activities. There must be something wrong with your clone.

**Pull request closed without merging**